src/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,19 @@
+.button {
+  @extend .btn;
+
+  &--white {
```

Comment:
@noor-ahmed All of the properties in here look not related to its class name 🤔
Should we move some properties which are not about color to another new class?
And we leave just only properties about color here 😅

Reply:
Created this file src/stylesheets/components/_submit_button.scss